src/wfutil.c

```diff
@@ -1571,6 +1571,7 @@ BOOL TypeAheadString(WCHAR ch, LPWSTR szT)
 {
    static DWORD tick64 = 0;
    static WCHAR rgchTA[MAXPATHLEN] = { '\0' };
+   static BOOL sameChar = TRUE;
    DWORD tickT;

```

Comment:
let's add a comment indicating that sameChar means that all of the characters in rgchTA thus far are the same.